### PR TITLE
Update the socket path to new location

### PR DESCRIPTION
**Pull request by hug-dev**
See [this issue](https://github.com/parallaxsecond/parsec/issues/250) for details.

### CODE REVIEW

File: src/core/ipc_handler/unix_socket.rs [hunk 1]
```diff
@@ -18,7 +18,7 @@ use std::io::{Error, ErrorKind};
 #[cfg(not(feature = "no-fs-permission-check"))]
 use std::os::unix::fs::MetadataExt;
 
-const DEFAULT_SOCKET_PATH: &str = "/tmp/parsec/parsec.sock";
+const DEFAULT_SOCKET_PATH: &str = "/run/parsec/parsec.sock";
```
Comment (hug-dev) [inline src/core/ipc_handler/unix_socket.rs:21]: There is already a way to modify the path of the socket at runtime, so for test I think we can change this path to `/tmp/parsec.sock` in the `e2e_tests` folder directly.

Comment (ionut-arm) [reply src/core/ipc_handler/unix_socket.rs:21]: There is? Wow, I had some good foresight there

**Review by ionut-arm — APPROVED** [review]
👌  let's DO THIS

**Comment by hug-dev** [comment]
I will wait for consensus on the three PRs before I start merging any of them 🚀 Just in case some changes is still needed here

**Pull request merged**